src/main/java/edu/ucla/library/sinai/handlers/LoginHandler.java

```diff
@@ -30,24 +31,43 @@ public LoginHandler(final Configuration aConfig, final JWTAuth aJwtAuth) {
     @Override
     public void handle(final RoutingContext aContext) {
         final MultiMap params = aContext.request().params();
+        final HttpServerResponse response = aContext.response();
 
         if (params.contains(TOKEN)) {
             final String rawToken = params.get(TOKEN);
-            final JWTOptions jwtOptions = new JWTOptions().setExpiresInSeconds(60);
-            final String token = myJwtAuth.generateToken(new JsonObject().put("sub", "asdf"), jwtOptions);
             final Cookie cookie = aContext.getCookie(TOKEN);
 
-            // Not perfect, we know, but a good faith effort
             if (cookie != null && rawToken != null && rawToken.equals(cookie.getValue())) {
-                myJwtAuth.authenticate(new JsonObject().put("jwt", token), authHandler -> {
-                    final HttpServerResponse response = aContext.response();
+                /**
```

Comment:
Because this file is so short, I didn't feel terribly guilty about adding these long comments, but I realize there's some repetition / redundancy between the comments. I can try to condense them if it feels like too much or is belaboring.

Reply:
Or: maybe I should condense them to a Javadoc on the `handle` method, then put line comments here instead? I may have answered my own question 🙃